po/meson.build

```diff
@@ -0,0 +1,6 @@
+i18n.gettext(meson.project_name(),
+  args: [
```

Comment:
Maybe we may want to unify the number of spaces to 4 among `meson.build` files?

Reply:
What do you mean exactly?

Reply:
Sorry for the confusion💦

We use 4 spaces for indentation in other `meson.build` files like `src/meson.build`. However, it seems only this `po/meson.build` file uses 2 spaces for indentation. So I think it would be even better to use 4 spaces for indentation also in this file.

Reply:
Hm yes I didn't noticed that. You're right. I think it's my editor that formatted it like that. I'll change that now.